source/_integrations/samsungtv.markdown

```diff
@@ -20,39 +20,33 @@ To add a TV to your installation without relying on the [discovery component](/i
 
 ```yaml
 # Example configuration.yaml entry
-media_player:
-  - platform: samsungtv
-    host: IP_ADDRESS
+samsungtv:
+  - host: IP_ADDRESS
+    name: My Samsung TV
```

Comment:
In the first example we want only required config variables, please remove everything that is optional here.
You can make a more advanced example above the `supported models` part 😉